lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -313,6 +313,9 @@ def filter_stream_from_files(tag, es)
     end
 
     def filter_stream_from_journal(tag, es)
+      if es.empty?
```

Comment:
ruby style - single line: `return es if es.empty?`

Reply:
@richm will do.  Did I correctly squash it into a single commit? Is the right way to do that as I did above or should i just submit a new clean PR with only that change?

Reply:
typically the way it works is that you use `git rebase -i origin/master` - keep the first commit, and fixup or squash the rest - then `git push -f frankreno master` - that will force push the rebased commit, which will also update this PR